FastLM-Backend/app.py

```
@@ -0,0 +1,1469 @@
+# 템플릿 카테고리 API
+@app.route('/api/template-categories', methods=['GET'])
+@jwt_required()
+def get_template_categories():
+    current_user_id = int(get_jwt_identity())
+    workspace_id = request.args.get('workspaceId')
+    
+    if workspace_id:
+        # 특정 워크스페이스의 카테고리 + 전역 카테고리
+        categories = NoticeCategory.query.filter(
+            (NoticeCategory.workspace_id == workspace_id) | 
+            (NoticeCategory.workspace_id == None)
```

Flake8 (7.2.0)

[error] comparison to None should be 'if cond is None:'

(E711)